### PR TITLE
Adding an Edit This Site link to sidebar

### PR DESCRIPTION
**Pull request by gbinal**
Both for our convenience and also to promote greater collaboration

### CODE REVIEW

File: _includes/sidebar.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
           <li><a href="{{ link.url }}">{{ link.text }}</a></li>
         {% endif %}
       {% endfor %}
+      <li>
+      <a href="https://github.com/GSA/https">Edit This Site</a>
```
Comment (alex) [inline _includes/sidebar.html]: I hate to be that guy, but should this be indented?

**Comment by alex** [comment]
Yay for encouraging contributions :sparkles: :cake: :sparkles:

File: _includes/sidebar.html (hunk 1 above)
Comment (gbinal) [inline _includes/sidebar.html]: No idea.  :P

Game to correct?

**Comment by gbinal** [comment]
Barring any complaints, I'd like to merge this before we share the site.

**Pull request merged**